src/offload/offload_library.c

```diff
@@ -39,6 +45,23 @@ const uint32_t colormap[] = {0xFFFFFF00,  // Yellow
                              0xFF0000FF,  // Blue
                              0xFF000080}; // Navy
 
+/*******************************************************************************
+ * \brief Initialize runtime.
+ * \author Rocco Meli
+ ******************************************************************************/
+void offload_init(void) {
+#ifdef __OFFLOAD_CUDA
```

Comment:
Nit, for symmetry:
```suggestion
#if defined(__OFFLOAD_CUDA)
```
?